src/main/java/seedu/address/model/recommendation/exceptions/DuplicateRecommendationException.java

```diff
@@ -0,0 +1,11 @@
+package seedu.address.model.recommendation.exceptions;
+
+/**
+ * Signals that the operation will result in duplicate Persons (Persons are considered duplicates if they have the same
+ * identity).
+ */
+public class DuplicateRecommendationException extends RuntimeException {
+    public DuplicateRecommendationException() {
+        super("Operation would result in duplicate persons");
```

Comment:
Just to double check, is this supposed to say "duplicate person"?

Reply:
Yea I changed it back already. It happened when I am creating the `UniqueRecommendationList`.